app/bundles/PointBundle/Event/GroupEvent.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\PointBundle\Event;
+
+use Mautic\CoreBundle\Event\CommonEvent;
+use Mautic\PointBundle\Entity\Group;
+
+class GroupEvent extends CommonEvent
```

Comment:
Is it necessary to extend the `CommonEvent`? I'd like to get rid of any unnecessary inheritance.
```suggestion
final class GroupEvent
```

Reply:
Probably not, I've used this class to be consistent. Let's remove it if it's not necessary.